cypress-tests/README.md

```diff
@@ -1,108 +1,46 @@
-# Cypress Tests for Renku
+# Cypress tests for Renku
 
-This repository aims to fill in gaps in current tests with Cypress.
+This section includes a set of acceptance tests for RenkuLab. The tests are
+written using [Cypress](https://www.cypress.io), run in Chrome, and the main target
+is the web interface of RenkuLab.
 
-# Test suites
+## Running the tests
 
-We currently have the following test suites
+You need a fully working RenkuLab deployment to run the tests against. You will also
+need a user account for the deployment.
 
-## R-Studio sessions
+If you wish to run the tests locally, you will need to clone the repository and have a
+recent version of Node.js installed. Then follow these steps:
 
-- Register a user / login
-- Create a R project
-- Launch a session
-- Open the session in the iframe view and test basic functionality
-- Stop the session
-- Delete the created project
+- Create a `cypress.env.json` file with the necessary user credentials. You can use the
+  `cypress.env.template.json` file as a template.
+- Install the dependencies with `npm install`.
+- Run the tests with `npm run e2e`. If you prefer not to use the GUI, you can run the
+  tests in headless mode with `npm run e2e:headless`.
 
-## Public Project
+> Tip: you might prefer not to save you password in plain text in the `cypress.env.json`
```

Comment:
I thought the section on how to run the tests with information passed in environment variables in general was useful and should be kept. Maybe including a table with the different ENV variables that are supported and their purpose.

Reply:
I added that in the last commit. I felt the names were self-explanatory, but having a table listing all the variables sounds useful anyway.